### ISSUE TITLE
VizDoom input visualizer

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEATv2#190]
Given all of the input parameters you made, we need a way of looking at what the network sees so we can get a feel for whether it makes sense. There should be a parameter called doomSeeInputs that creates a window that shows the portion of the screen that the agent actually sees. This should be straight-forward if the smudge factor is turned off, since you are just copying that exact portion of the screen into an image. It's more interesting if you are smudging, since you will be displaying the smudged values.

Also take into account the color that the input is using. If we are only looking at red, then only use the red values in the input window.

**Comment by Serenamidori** [comment]
I've added the method showInputs() to VizDoomTask, and when it's called it will show the inputs that the agent is seeing. The smudge factor will just show us how "big" the smudges are, so the height and width are consistent with their original parameters.

I also added a new Stat called MostExtreme which takes a double array and finds the entry that's the most distinct from the others- that is, it finds the one with the biggest difference from the other. :ok_hand: 

We've tested both, and I've got the inputs for the parameters that could best optimize VizDoom Scenarios when evolving.
